apps/web/src/mutations/use-experiment-create-exercise-submission-mutation.ts

```diff
@@ -13,8 +13,14 @@ const mutation = gql`
   }
 `
 
-export function useCreateExerciseSubmissionMutation() {
+export function useCreateExerciseSubmissionMutation(path: string) {
   const mutationFetch = useMutationFetch()
+  const splitPath = path.split('/').filter((p) => p.length > 0)
+  const experimentIds = [30680, 23869, 66809]
+  const pathContainsExperimentId = splitPath.some((p) =>
+    experimentIds.includes(+p)
+  )
```

Comment:
maybe something like this would also work?
``` ts
const isExperimentPath = ['/30680', '/23869', '/66809'].find((slug) => path.includes(slug))
```

Reply:
or
``` ts
const isExperimentPath = [30680, 23869, 66809].find((id) => path.includes(`/${id}`))
```

Reply:
Thanks, done